foundations/foundation-metrics/src/main/java/org/apache/servicecomb/foundation/metrics/publish/DefaultHealthCheckExtraData.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.servicecomb.metrics.common;
+package org.apache.servicecomb.foundation.metrics.publish;
 
 public class DefaultHealthCheckExtraData {
```

Comment:
What's the relationship between the DefaultHealthCheckExtraData and HealthCheckResult.extraData?
We don't need to use the String to store result,  it's easy to use toString method to turn the instance of HealthCheckExtraData into a string.

Reply:
We may no need provider this default health check because :
1.make some  confusion
2.Service Center had same function

Reply:
Had removed